<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gilgamesh</class>
 <widget class="QMainWindow" name="Gilgamesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1644</width>
    <height>1088</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gilgamesh Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="ShotTableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>731</width>
      <height>691</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>800</y>
      <width>741</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Messaging</string>
    </property>
    <widget class="QTextEdit" name="logWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>501</width>
       <height>181</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="contentWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="Plotwidget" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>300</y>
      <width>871</width>
      <height>451</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>861</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hLayout"/>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Criterion:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="criterionEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>60</y>
      <width>256</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>60</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="listActions">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="actionButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>770</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>List of signals available</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Testbed configuration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Component properties</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>800</y>
      <width>451</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>780</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Program description</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>ShotTableView</zorder>
   <zorder>contentWidget</zorder>
   <zorder>Plotwidget</zorder>
   <zorder>label</zorder>
   <zorder>criterionEdit</zorder>
   <zorder>updateButton</zorder>
   <zorder>treeView</zorder>
   <zorder>textEdit</zorder>
   <zorder>listActions</zorder>
   <zorder>actionButton</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1644</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionManage_programs"/>
    <addaction name="actionManage_components"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSet_Gas"/>
    <addaction name="actionAdd_WinCC_data"/>
    <addaction name="actionAdd_Generator_data"/>
    <addaction name="actionSet_Parameters"/>
    <addaction name="actionSet_Manual_Position"/>
    <addaction name="actionConvert_to_Real_Pos"/>
    <addaction name="actionCalculate_density"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionReinitialize_Shot_database"/>
    <addaction name="actionImport_Mapping_from_CSV"/>
    <addaction name="actionExport_Mapping_from_CSV"/>
    <addaction name="actionReload_Signal_Database"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuProgram"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAdmin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionManage_programs">
   <property name="text">
    <string>Manage programs</string>
   </property>
  </action>
  <action name="actionManage_components">
   <property name="text">
    <string>Manage components</string>
   </property>
  </action>
  <action name="actionSet_Gas">
   <property name="text">
    <string>Set Gas</string>
   </property>
  </action>
  <action name="actionAdd_WinCC_data">
   <property name="text">
    <string>Add WinCC data</string>
   </property>
  </action>
  <action name="actionAdd_Generator_data">
   <property name="text">
    <string>Add Generator data</string>
   </property>
  </action>
  <action name="actionReinitialize_Shot_database">
   <property name="text">
    <string>Reinitialize Shot database</string>
   </property>
  </action>
  <action name="actionImport_Mapping_from_CSV">
   <property name="text">
    <string>Import Mapping from CSV</string>
   </property>
  </action>
  <action name="actionSet_Parameters">
   <property name="text">
    <string>Set Parameters</string>
   </property>
  </action>
  <action name="actionReload_Signal_Database">
   <property name="text">
    <string>Reload Signal Database</string>
   </property>
  </action>
  <action name="actionSet_Manual_Position">
   <property name="text">
    <string>Set Manual Position</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="actionExport_Mapping_from_CSV">
   <property name="text">
    <string>Export Mapping from CSV</string>
   </property>
  </action>
  <action name="actionConvert_to_Real_Pos">
   <property name="text">
    <string>Convert to Real Pos</string>
   </property>
  </action>
  <action name="actionCalculate_density">
   <property name="text">
    <string>Calculate density</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
